.github/workflows/docs.yml

```diff
@@ -55,21 +55,32 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: ./.github/actions/setup-java
       - uses: ./.github/actions/setup-node
       - uses: ./.github/actions/setup-python
       - run: |
-          uv pip install --system .. -r ../requirements/doc-requirements.txt
+          uv pip install --system pytest pytest-cov plotly -r ../requirements/doc-requirements.txt
+          uv pip install --system -e '..[gateway]'
       - run: |
           yarn install --immutable
+      - uses: ./.github/actions/show-versions
+      - run: |
+          yarn convert-notebooks
       - name: Build docs
+        env:
+          GTM_ID: "GTM-TEST"
+          API_REFERENCE_PREFIX: https://pr-${{ github.event.pull_request.number }}--mlflow-docs-preview.netlify.app/latest/
```

Comment:
This should be `https://pr-${{ github.event.pull_request.number }}--mlflow-docs-preview.netlify.app/docs/` I think.